### PR TITLE
fix(AOC): prevent early clearing of OOOI-Data #8369

### LINKED ISSUE #8369: INIT/REVIEW times resetting at touchdown
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-01-02T11:57:34+00:00",
    "ref": "refs/heads/master",
    "sha": "1f16d82017a1b163a6c4de4db6c7092d6b175ff8",
    "actor": "Benjozork",
    "event_name": "manual",
    "pretty_release_name": "master:1f16d820",
    "version": "v0.12.0-dev.1f16d82"
}
```

### Describe the bug

INIT/REVIEW times resetting at touchdown. After takeoff there are two identical times for OFF and ON fields. ON ground time is equal take OFF time probably because of micro jumps during takeoff roll. IN field is never filled. The whole page resets to empty exactly at touchdown moment. See pictures attached 

### Expected behavior

Some say this page should be reset at next flight. I don't know if it's possible. But it should not resets before full power down or exit a flight.

### Steps to reproduce

1. Do a full flight and look at INIT/REVIEW page

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

Vitold69

### PR DESCRIPTION
**Pull request by cptnuss-ops**
Fixes #8369 

## Additional context

Discord username (if different from GitHub):

### Simbrief flow
1. Prepare Aircraft for flight, depart and land
2. Shutdown at gate and check OOOI times are still present
3. Prepare next flight and pushback to init new cycle
4. Observe OOOI-Times on AOC-REVIEW page are cleared and a new leg is started

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Review by Benjozork — APPROVED** [review]

**Comment by alepouna** [comment]
Hey heads up, you have conflicting files 🙌

**Comment by cptnuss-ops** [comment]
> Hey heads up, you have conflicting files 🙌

Sorry, my bad. I resolved the conflict.

**Comment by alepouna** [comment]
No worries what so ever, thank you for quick patch, I will add RTT labels so it can get QA'd!

**Comment by noonebling** [comment]
Quality Assurance Trainee Report

Discord          : noonebling
Object of testing: 8665
Tier of Testing  : 1
Date             : 06/07/2024

Testing Process: 
1. Loaded up flight, took off and landed
2. Shutdown at gate and checked OOOI times
3. Prepped next flight and pushbacked to init new cycle
4. Checked if OOOI times are cleared and new leg is started



Negatives:
N/A

Testing Results: 
Passed

Conclusions: 
Works as expected

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_ATSU.js [hunk 1]
```diff
@@ -111,3 +111,232 @@ const getSimBriefOfp = (mcdu, updateView, callback = () => {}) => {
             updateView();
         });
 };
+
+/**
+ * There are two uplink requests that are made at the same time:
+ * - AOC ACT F-PLN
+ * - PERF DATA
+ */
+const insertUplink = (mcdu) => {
```
Comment (Maximilian-Reuter) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_ATSU.js:120]: This code seems to be from a bad merge or something. As that code has now all be relocated to typescript.
I think you will have to redo your changes on a branch that is clean from master. 

Simbrief uplink is now all happening here: https://github.com/flybywiresim/aircraft/blob/2cdf866fb140b72151c25e2413dd5365364ba08f/fbw-a32nx/src/systems/fmgc/src/flightplanning/new/uplink/SimBriefUplinkAdapter.ts#L115

**Review by Maximilian-Reuter — CHANGES_REQUESTED** [review]

**Comment by cptnuss-ops** [comment]
Ok, I'll close this PR and start again with a fresh codebase.

**Pull request closed without merging**